pyproject.toml

```diff
@@ -40,3 +41,6 @@ build-backend = "poetry.core.masonry.api"
 [tool.black]
 line-length = 120
 include = 'encord\/.*'
+
+[tool.isort]
+profile = "black"
```

Comment:
What is the link between isort and black?

Reply:
I've added a comment to explain.